ios/facebook_app_events.podspec

```
@@ -18,9 +18,9 @@ Flutter plugin for Facebook Analytics and App Events
 
   # Do not specify PATCH version of FBSDKCoreKit. See README file for explanation
   # https://github.com/oddbit/flutter_facebook_app_events#dependencies-on-facebook-sdk
-  s.dependency 'FBSDKCoreKit', '~> 16.1'
+  s.dependency 'FBSDKCoreKit', '~> 16.3'
   
   # See docs on FBAudienceNetwork
   # https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/add-sdk/
-  s.dependency 'FBAudienceNetwork', '~> 6.12'
+  s.dependency 'FBAudienceNetwork', '~> 6.14'
```

```suggestion
  s.dependency 'FBAudienceNetwork', '~> 6.15'
```

According to [Meta Docs](https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/add-sdk/)
they released on March 15, 2024 v-6.15